src/components/TaxPicker.tsx

```diff
@@ -1,16 +1,28 @@
-import lodashGet from 'lodash/get';
 import React, {useMemo, useState} from 'react';
-import _ from 'underscore';
-import OptionsSelector from '@components/OptionsSelector';
+import type {EdgeInsets} from 'react-native-safe-area-context';
 import useLocalize from '@hooks/useLocalize';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as OptionsListUtils from '@libs/OptionsListUtils';
 import * as TransactionUtils from '@libs/TransactionUtils';
 import CONST from '@src/CONST';
-import {defaultProps, propTypes} from './taxPickerPropTypes';
+import OptionsSelector from './OptionsSelector';
 
-function TaxPicker({selectedTaxRate, policyTaxRates, insets, onSubmit}) {
+type TaxPickerProps = {
+    /** Collection of tax rates attached to a policy */
+    policyTaxRates?: OptionsListUtils.PolicyTaxRateWithDefault;
```

Comment:
```suggestion
    policyTaxRates: PolicyTaxRateWithDefault;
```

Import type separately and make the prop required (then remove default value)